tests/unit/multi-currency/test-class-frontend-currencies.php

```diff
@@ -77,28 +77,32 @@ public function test_get_woocommerce_currency() {
 	public function test_get_price_decimals_returns_default_settings() {
 		$current_currency = new WCPay\MultiCurrency\Currency( 'RANDOM_CODE' );
 		$this->mock_multi_currency->method( 'get_selected_currency' )->willReturn( $current_currency );
+		$this->mock_locale->method( 'get_currency_format' )->willReturn( $this->mock_get_currency_format( $current_currency ) );
```

Comment:
We could potentially add the expectation here (and other places we mock this function) that this method will only get called once in the context of this test, which makes the test more robust by ensuring `get_currency_format` only gets called the amount of times we expect it to get called.

For example:

```php
$this->mock_locale->expects( $this->once() )
    ->method( 'get_currency_format')
    ->willReturn( $this->mock_get_currency_format( $current_currency );
```